src/main/java/storagesystem/model/Location.java

```diff
@@ -14,6 +20,11 @@ public Location(String name, String description, Image image) {
         this.image = image;
     }
 
+    public Location(){
+        this.name = "Hasen";
+        this.description = "Open the door";
```

Comment:
Should be sent in as parameters instead.

Reply:
fixed in new commit